razorpages/RazorPagesFeckMelzer.md

```diff
@@ -0,0 +1,274 @@
+﻿# *Razor Pages ASP.NET Core*
+
```

Comment:
Ein durchgängiges Beispiel darstellen (Razor-Page + PageModel)!
Herausarbeiten, dass Informationen, welche an den Server transportiert werden sollen in <input> Tag innerhalb eines <form>-Tags sich befinden müssen (hier TagHelper einbauen).
Screenshot des Http-Requests in welchem die Werte übertragen werden noch dazunehmen.